ambry-tools/src/main/java/com/github/ambry/tools/admin/ServerAdminTool.java

```diff
@@ -329,6 +332,16 @@ public static void main(String[] args) throws Exception {
               config.getOption, bResponse.getFirst());
         }
         break;
+      case BlobIndex:
+        blobId = new BlobId(config.blobId, clusterMap);
+        Pair<ServerErrorCode, String> biResponse = serverAdminTool.getBlobIndex(dataNodeId, blobId);
+        if (biResponse.getFirst() == ServerErrorCode.No_Error) {
+          LOGGER.info("Blob index values of {} from {} is {}", blobId, dataNodeId, biResponse.getSecond());
```

Comment:
Nit: In case the index response string is a serialized json, would it be helpful for users to the tool if response is  [indented](https://stackoverflow.com/a/14532435).

Reply:
shouldn't be an issue, this should be an admin tool. Anybody uses this can format the result using other tools